### ISSUE TITLE
Not compatible with @quasar/app v2.0.1

### ISSUE THREAD

**Issue by codenamezjames** [freddy38510/quasar-app-extension-ssg#12]
App · ⚠️  Extension(ssg): is not compatible with @quasar/app v2.0.1. Required version: ^1.5.2

I get this error whenever I run `quasar ext invoke ssg`

```App · Invoking "ssg" Quasar App Extension

? Enable SPA Fallback: Yes
? Set the path to the SPA fallback: 404.html
? Enable CriticalCss: Yes
? Preload strategy: default

 App · Updating /quasar.extensions.json for "ssg" extension ...
 App · Running App Extension install script...
 App · ⚠️  Extension(ssg): is not compatible with @quasar/app v2.0.1. Required version: ^1.5.2```

**Comment by freddy38510** [comment]
I didn't test the AE with @quasar/app above 1.5.2 version and @quasar above 1.8.3 version.

If you want to try with latest version of quasar, i just created a new branch [without semver restrictions](https://github.com/freddy38510/quasar-app-extension-ssg/commit/95bd492e17a7e9a0dbc116ab1b71b554fbac5b4d).

Just add the app extension dependency like this `yarn add --dev https://github.com/freddy38510/quasar-app-extension-ssg#develop`.

Please, let me know if latest version of quasar breaks compatibility with this AE, thanks.

**Comment by jcdb95** [comment]
Hey, i am using v2.0.5 of quasar and it worked by running `npm install https://github.com/freddy38510/quasar-app-extension-ssg#develop`

**Comment by freddy38510** [comment]
Fixed in 2e2f693222ee3a597892d264d3911e5ebbb05bd7. You should now use the master branch.
Thanks for your feedbacks both of you @jcdb95, @codenamezjames.
